### PR TITLE
add basename option

### PR DESCRIPTION
**Pull request by dnoneill**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -48,7 +48,8 @@ options = {
   columns: 4,
   group: 20,
   gif: false,
-  keep_images: false
+  keep_images: false,
+  basename: 'filename' #this is optional. If not specificed this will be 'sprite'
```
Comment (jronallo) [inline README.md]: I'd prefer this line be changed. So it should instead be:

```
basename: 'sprites', # Base filename for outputs. Optional; default is "sprites".
```

Can you please make this change?

**Comment by jronallo** [comment]
Is there now also a CLI option for setting the basename? Could that be added to be consistent?

File: README.md (hunk 1 above)
Comment (dnoneill) [reply README.md]: @jronallo Done

**Comment by dnoneill** [comment]
@jronallo I am not sure I understand this comment. Could I get more clarification.

**Comment by jronallo** [comment]
The project can be used as either a library or via the CLI. Is this option to set the basename also available in the CLI? The CLI example in the readme should be updated to reflect this new option.

**Comment by dnoneill** [comment]
@jronallo for some reason I kept missing the examples in the README. I have updated the gem and docs to include CLI usage.

**Comment by jronallo** [comment]
@dnoneill Thank you.

**Pull request merged**